scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -37,6 +38,15 @@ function prepare_vm {
 	    printf "\n kernel-${kernel_version} installation failed \n"
     fi
     yum -y install kernel-devel-${kernel_version}
+
+    yum -y install yum-utils
+    yumdownloader -y kernel-${kernel_version} kernel-devel-${kernel_version}
+
+    wget https://aka.ms/downloadazcopy-v10-linux
+    tar xvf downloadazcopy-v10-linux
+    for file in ./*.rpm; do
+        ./azcopy_linux_amd64_10.1.2/azcopy cp "$(basename "$file")" "https://lisbuildstorage04.blob.core.windows.net/kernel/${storage_sas_token}" --recursive
```

Comment:
again, the path can be change. a better approach?

Reply:
Is any place to cleanup old rpm packages which upload here to https://lisbuildstorage04.blob.core.windows.net/kernel?

Reply:
it is not implemented yet